bluebutton-data-pipeline-rif/src/main/java/gov/hhs/cms/bluebutton/datapipeline/rif/model/InpatientClaimGroup.java

```diff
@@ -213,9 +213,9 @@
 	public List<IcdCode> diagnosesExternal = new LinkedList<>();
 
 	/**
-	 * Represents the data contained in {@link Column#LINE_NUM} and subsequent
-	 * columns: one entry for every "claim line" in the claim represented by
-	 * this {@link InpatientClaimGroup} instance.
+	 * Represents the data contained in {@link Column#CLM_LINE_NUM} and
+	 * subsequent columns: one entry for every "claim line" in the claim
+	 * represented by this {@link InpatientClaimGroup} instance.

```

Comment:
Okay, had a feeling you might be a bit confused about it -- glad I asked. Everything in Git occurs on branches, and `master` is just another branch. It's best to think of the branches as separate streams, that merge and diverge every so often. In fact, that's how most GUIs visualize them. Take a look here and see if you can figure out what's going on: <https://github.com/HHSIDEAlab/bluebutton-data-pipeline/network>. Can you find your earlier JavaDoc fix branch in there, and the branch for this pull request?

So. Your branch has diverged from `master`, but now needs some changes that are in `master`. With Git, you've got two options for how to accomplish that: rebasing or merging.

Rebasing would "go back in time" and alter the history of the branch, making it look as if the branch "started" later than it actually did. This is great! Except you've shared this branch with other folks (me and GitHub), and anything we've done with it will be borked. In this case, I'd expect that you'd lose all of the line-level comments on this PR. Sometimes, that might be the correct move. Probably isn't here, though. If you tried this, Git would warn you by making you add a `force` flag somewhere in your push to make it happen.

Merging doesn't "go back in time" like that; it just selectively copies all of the commits your branch is missing and pastes them into it. You'd accomplish this, like follows:

    # Let's assume you're starting on your branch, cbbx-42-foo.
    
    # Do you have any uncommitted changes?
    $ git status
    
    # If so, let's first stash them, so we can play around without losing them.
    $ git stash save "My uncommitted work on cbbx-42-foo"

    # We want to go grab those latest changes on master that we'd like to merge in, so first switch to master and make sure it's up to date.
    $ git checkout master
    $ git pull
    
    # Now, we want to switch back to your out-of-date branch and merge the latest from master into it.
    $ git checkout cbbx-42-foo
    $ git merge master
    $ git push
    
    # Now, just pop your changes back out of the stash and you're good to go.
    $ git stash pop

If there aren't any conflicts, all you'll have to do here is confirm or modify the auto-generated commit message for the merge. If there are conflicts, the merge process will walk you through resolving them, though you may want to Google or ask me for me some help. It's tricky the first few times, until you get your head wrapped around it.

That all make sense?